dll/shellext/ntobjshex/resource.h

```diff
@@ -11,6 +11,8 @@
 
 #define IDS_NTOBJFLD_NAME        101
 #define IDS_REGISTRY_NAME        102
+#define IDS_REGISTRY_COMMENT     103
+#define IDS_NTOBJFLD_COMMENT     104
```

Comment:
Please sort the values (and translations above) in the natural order they were implemented (first, the NTOBJ stuff, then, the registry stuff).

Reply:
OK